Clinical-Genomics/cg
# Lists organisms by internal_id in the Orderportal

This PR lists the organisms in the order portal in alphabetical order

How to test:
0. install on stage (add instructions on how to as needed)

1. login to order portal
1. create an Microbial order
1. add sample
1. open Organism drop down

Expected outcome:
The organisms should be listed in alphabetical order
